simple.css

```diff
@@ -947,27 +947,60 @@ etym, re {
   text-decoration: underline;
 }
 
+#verse-popup {
+  visibility: hidden;
+}
+
 #verse-popup.visible {
   background: var(--bg);
   border: 1px solid var(--border);
-  border-radius: 5px;
+  border-radius: 5px 5px 0 0;
   bottom: 0;
   color: var(--text);
-  padding: 10px;
+  font-size: 75%;
   position: sticky;
+  visibility: visible;
+}
+
+#verse-popup .words {
+  padding: .5em;
 }
 
-#verse-popup span {
+#verse-popup.minimized .words {
+  display: none;
+}
+
+#verse-popup .words .word {
   display: inline-block;
   cursor: pointer;
   padding: .15em;
 }
 
-#verse-popup span:hover {
+#verse-popup .word:hover {
   background-color: var(--accent-bg);
 }
 
-#verse-popup span.highlighted {
+#verse-popup .word.highlighted {
   color: var(--accent);
   font-weight: bold;
+}
+
+#verse-popup .bottomBar {
+  display: flex;
+  flex-direction: row;
+  text-align: center;
```

Comment:
We should probably add `padding: .25em;` to make it match the "words" div padding.